django/contrib/admin/widgets.py

```diff
@@ -194,7 +195,22 @@ def label_for_value(self, value):
         key = self.rel.get_related_field().name
         try:
             obj = self.rel.model._default_manager.using(self.db).get(**{key: value})
-            return '&nbsp;<strong>%s</strong>' % escape(Truncator(obj).words(14, truncate='...'))
+            text = escape(Truncator(obj).words(14, truncate='...'))
```

Comment:
This should be outside the try block as it's not expected to raise the exception.